n3fit/src/n3fit/layers/rotations.py

```diff
@@ -100,18 +100,20 @@ class AddPhoton(MetaLayer):
     output it is set to its real value.
     """
 
-    def __init__(self, output_dim=14, **kwargs):
+    def __init__(self, pdf_ph, output_dim=14, **kwargs):
```

Comment:
I would remove the default, let's pass it from external every time:
```suggestion
    def __init__(self, pdf_ph, output_dim, **kwargs):
```

Reply:
You're right, but this wouldn't be the only place in n3fit..